pints/tests/test_plot.py

```diff
@@ -12,6 +12,7 @@
 import unittest
 import numpy as np
 import matplotlib
+import matplotlib.pyplot as plt
```

Comment:
This needs to go after `matplotlib.use`

Reply:
OK! Yes, I got flake8 complained so I moved it before... Yes, I can do #noqa and move it later!